### ISSUE TITLE
Expand number years included in OREx and New Horizons tasks

### ISSUE THREAD

**Issue by jordanpadams** [NASA-PDS/pds-report-service#9]
## 💡 Description

1) for NH, back to 2006, or alternatively, at least back to 2015, the Pluto Encounter Date, the total number of files downloaded of NH mission data, the total volume of NH mission data, and the total number of users, per month, from SBN? If possible, could you do the same for the holdings in RMS downloads of the NH data as well? 

2) for O-REx, back to September, 2016 (launch), can you provide the same... monthly tallies of the total number of files downloaded of O-REx mission data, the total volume of O-REx mission data, and the total number of users for the SBN-PSI O-REx data holdings?

**Comment by c-suh** [comment]
Have started update for SBN database.

**Comment by c-suh** [comment]
@jordanpadams a status:

1. I've generated reports and put together the information for
- SBN New Horizons from 2012
- SBN OREx from 2019

2. RMS New Horizons config has been updated, a database rebuild was tested with one month of data, and reports generated with the desired information. A full update is happening with the rest of the data, which will take a few days, after which it'd be up to another 24 hours to generate all the reports.

Go ahead and send the information for SBN?
